CHANGELOG.md

```diff
@@ -1,3 +1,28 @@
+## v1.17.0 (09/10/2023)
+
+**Common**
+* Improved support of HTTP proxy on iOS.
+* Cleaned up DNS cache on error in cURL (Android).
+* Updated the `olp-cpp-sdk-core/CMakeLists.txt` to include the externals include directory.
+* Added the possibility to specify DNS using `WithDNSServers` to `olp::http::NetworkSettings` API.
+* Improved logging in `olp::client::PendingUrlRequests`.
+* Added `CallApiStream` to `olp::client::OlpClient` API, which is using a `SAX` parser and a `JSON` byte stream to parse the layer partitions while they are downloading.
+* Fixed the `boost` and `rapidjson` includes path resolution in `olp-cpp-sdk-core/CMakeLists.txt`.
+* Changed log level to `WARNING` for error message when max number of `HTTP` requests has been reached in `olp::http::SendOutcome OLPNetworkIOS`.
```

Comment:
```suggestion
* Changed the log level to `WARNING` in the error message that is returned when the maximum number of `HTTP` requests is reached in `olp::http::SendOutcome OLPNetworkIOS`.
```

Reply:
Log level is the way the message is shown, not something in the message.